src/ToggleButton.svelte

```diff
@@ -0,0 +1,72 @@
+<script lang="ts" context="module">
+  import { capitalize } from "@app/utils";
+
+  export interface ToggleButtonOption<T> {
+    count?: number;
+    title?: string;
+    value: T;
+  }
+</script>
+
+<script lang="ts" strictEvents>
```

Comment:
nice `strictEvents`! learning every day something new 😎

Reply:
Yeah, me too. We were all waiting for typed events and generics in components, and now they're finally here, albeit still experimental. 🎉